src/app/legacy/components/RadioSchedule/ScheduleItemHeader/index.jsx

```diff
@@ -113,12 +98,18 @@ const ScheduleItemHeader = ({
     isRelative: false,
   });
 
+  const listenLabel = listenLabelTranslations[state];
+
   const content = (
+    // VisuallyHiddenText outside of the Live Label is only present if the state is not live
+
     // This is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
     // eslint-disable-next-line jsx-a11y/aria-role
     <span role="text" id={`scheduleItem-${id}`}>
-      <VisuallyHiddenText>{`${listenLabelTranslations[state]}, `}</VisuallyHiddenText>
-      {isLive && <LiveLabel liveText={liveLabel} ariaHidden />}
+      {listenLabel && (
```

Comment:
We can move this comment right next to the code?

```suggestion
    // This is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
    // eslint-disable-next-line jsx-a11y/aria-role
    <span role="text" id={`scheduleItem-${id}`}>
    // VisuallyHiddenText outside of the Live Label is only present if the state is not live
      {listenLabel && (
```

Reply:
fixed in vscode